CHANGELOG.md

```diff
@@ -1,3 +1,103 @@
+# v1.8.0 (2022-06-08)
+
+## OS Changes
+* Add support for Kubernetes 1.23 variants ([#2188])
+* Add aws-ecs-1-nvidia variant with Nvidia driver support ([#2128], [#2100], [#2098], [#2167], [#2097], [#2090], [#2099])
```

Comment:
Do we want to consider adding sub-sections, like `General`, `ECS-specific`, `Kubernetes-specific`? Different readers will find different features exciting, and right now they're all mixed together.